web/src/views/DandisetLandingView/DandisetLandingView.vue

```diff
@@ -129,7 +129,7 @@
                     {{ k }}
                   </v-subheader>
                   <v-list-item
-                    :key="k"
+                    :key="`${k}-item`"
```

Comment:
One more annoying question :smiley: 

Why not keep this as-is? Adding the `-item` doesn't do anything to improve the keying.

Reply:
The `v-subheader` above uses `k` as a key, so keeping it as is results in duplicate keys.